src/apps/competitions/models.py

```diff
@@ -50,6 +51,8 @@ class Competition(ChaHubSaveMixin, models.Model):
     # we use filed type to distinguish 'competition' and 'benchmark'
     competition_type = models.CharField(max_length=128, choices=COMPETITION_TYPE, default=COMPETITION)
 
+    fact_sheet = JSONField(blank=True, null=True, max_length=4096)
```

Comment:
```suggestion
    fact_sheet_questions = JSONField(blank=True, null=True, max_length=4096)
```

think that's too verbose?

Reply:
Maybe, will there be any other kind of `fact_sheet` we will need to differentiate it from?